app/models/use_cases/create_diagnosis.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module UseCases
+  class CreateDiagnosis
+    class << self
+      def create_with_params(params)
+        diagnosis = Diagnosis.create visit_id: params['visit_id']
+        return nil unless params['diagnosed_needs']
```

Comment:
It will eventually. (if one add general comments to the visit. Those comments can be useful even if no formal needs were specified)